match/lib/match/utils.rb

```diff
@@ -9,9 +9,18 @@ def self.import(item_path, keychain)
       # as the keychain path.
       #
       # We need to expand each path because File.exist? won't handle directories including ~ properly
+      #
+      # We also try to append `-db` at the end of the file path, as with Sierra the default Keychain name
+      # has changed for some users:
+      # - https://github.com/fastlane/fastlane/issues/5649
```

Comment:
I personally think it's good to provide some context and background on why a certain line or method was added. This way, future fastlane team can see why some things were added, including the complete discussion of why this change is necessary, with an easy way to actively reach out to people who had this problem. 
I agree with you that 3 issues might be too many, we can reduce it to the most important one, if you prefer that 👍